### ISSUE TITLE
A11y_VSCode_ViewExtensions_Screenreader:NVDA is not reading the content and focus goes to install button but it does not read as install button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52001]
**Environment Details**:
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**

ScreenReader,NVDA[2018.2],JAWS[18.0.5038],MAS1.3.1
Repro Steps:
1. Launch VS Code.
2. Navigate to View(Alt+V) menu option and select "Extensions" from dropdown.
3. Navigate to tree view under the search box.
4. Check what NVDA is reading.

Actual:
 NVDA reads only level when focus goes to install button.
Same issue observed in Jaws also.
Expected:
NVDA should read the install button though focus goes to install button.
Recommendations:
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
User will not able to know what is written on the screen.
**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)
Does this issue occur when all extensions are disabled?: Yes
**Attachment For Reference:**
![a11y_vscode_viewextensions_screenreader_nvda](https://user-images.githubusercontent.com/39204909/41461230-bf7e49a8-70ac-11e8-9bb7-961c36df9451.png)
